junixsocket-native/src/main/c/org_newsclub_net_unix_NativeUnixSocket.c

```diff
@@ -428,6 +440,9 @@ JNIEXPORT void JNICALL Java_org_newsclub_net_unix_NativeUnixSocket_close
 (JNIEnv * env, jclass clazz, jobject fd) {
 	int handle = org_newsclub_net_unix_NativeUnixSocket_getFD(env, fd);
 	int ret = close(handle);
+	while(ret == -1 && errno == EINTR) {
```

Comment:
close shouldn't be tried again (due to file descriptor re-use)

Reply:
Great idea!
see this commit https://github.com/kohlschutter/junixsocket/commit/45231ddca692299c2715878082f5b3748a5d4336
for the current implementation.

Instead of using a while-loop, let's use a do-while. This avoids duplication.